src/containers/connect/ConnectCTA.js

```diff
@@ -17,16 +17,26 @@ const CTAWrapper = styled.div`
   display: flex;
   flex: 0 0 auto;
   font-size: 15px;
+  justify-content: center;
   line-height: 18px;
+  margin: 0 0 30px 0;
+
+  @media only screen and (min-width: 1020px) {
```

Comment:
1. Should be the constant from `core/style/Sizes.js`
2. I don't think this actually does anything. What did you intend this to do?

Reply:
Used the constant. 

It maintains the dimension that we originally had at the larger size, rather than add 30px. The CTA buttons remain better aligned vertically, e.g. bottom CTA button is flush with the bottom of the 'Let's work together' subtitle.

Reply:
And the reason why 30px is added at smaller screen is to add space between the two buttons when they're stacked vertically.